runtime/vm/jfr.cpp

```diff
@@ -965,16 +965,16 @@ jfrCPULoad(J9VMThread *currentThread)
 				jfrEvent->jvmSystem = 0;
 			} else {
 				int64_t timeDelta = currentTime - jfrState->prevProcTimestamp;
-				jfrEvent->jvmUser = OMR_MIN((currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / ((double)numberOfCpus * timeDelta), 1.0);
-				jfrEvent->jvmSystem = OMR_MIN((currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / ((double)numberOfCpus * timeDelta), 1.0);
+				jfrEvent->jvmUser = OMR_MIN((currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / ((float)numberOfCpus * timeDelta), 1.0f);
+				jfrEvent->jvmSystem = OMR_MIN((currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / ((float)numberOfCpus * timeDelta), 1.0f);
```

Comment:
`OMR_MIN` evaluates one of its arguments twice; suggest using a local to avoid that.